package.json

```diff
@@ -32,8 +32,8 @@
   "homepage": "https://github.com/davidtheclark/react-aria-modal#readme",
   "dependencies": {
     "focus-trap-react": "^3.0.3",
-    "no-scroll": "^2.0.0",
-    "react-displace": "^2.1.4"
+    "no-scroll": "^2.1.0",
+    "react-displace": "^2.2.1"
```

Comment:
These semver changes should be necessary as part of this PR, right? They shouldn't really be necessary at all, since this module is still compatible with the other older versions.